metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/RegistryUtil.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+package com.netflix.metacat.common.server.util;
+
+import com.netflix.spectator.api.Registry;
+
+import java.util.concurrent.ThreadPoolExecutor;
+
+/**
+ * Utility functions for Registry.
+ * @author amajumdar
+ */
+public final class RegistryUtil {
+    private static final String NAME_MONITORED_POOL = "MonitoredThreadPool_";
```

Comment:
This is not specific to ThreadPool. We could add more helper functions. Based on Toms suggestion, I will update the method name.